tools/setup/postgresql-init-dev-db

```diff
@@ -116,6 +119,8 @@ SELECT unnest(
         ARRAY['ALTER SYSTEM SET pgroonga.enable_wal = ''on''', 'SELECT pg_reload_conf()']
     END
 ) \gexec
+
+CREATE EXTENSION pg_search;
```

Comment:
Note that we recommend also installing `vector` (pgvector) if you want to do hybrid search: https://docs.paradedb.com/documentation/guides/hybrid